app/notification_email.go

```diff
@@ -171,7 +173,9 @@ func (a *App) getNotificationEmailBody(recipient *model.User, post *model.Post,
 	var bodyPage *utils.HTMLTemplate
 	if emailNotificationContentsType == model.EMAIL_NOTIFICATION_CONTENTS_FULL {
 		bodyPage = a.NewEmailTemplate("post_body_full", recipient.Locale)
-		bodyPage.Props["PostMessage"] = a.GetMessageForNotification(post, translateFunc)
+		postMessage := a.GetMessageForNotification(post, translateFunc)
+		normalizedPostMessage := a.generateHyperlinkForChannels(postMessage, teamName, teamURL)
+		bodyPage.Props["PostMessage"] = template.HTML(normalizedPostMessage)
```

Comment:
What happens if someone sends a message that looks like `<b>Hello, world!</b>`? Does that text get bolded? If so, we'll need to sanitize any HTML in the post before passing it into `generateHyperlinkForChannels`

Reply:
Should I not be using template.HTML or should I just escape the postMessage?